core/setup.py

```diff
@@ -43,9 +43,7 @@
     include_package_data=True,
     test_suite="test",
     entry_points={
-        "console_scripts": [
-            "dbt = dbt.main:main",
-        ],
+        "console_scripts": ["dbt = dbt.main:main", "dbt-cli = dbt.cli.main:cli"],
```

Comment:
Maybe we should leave this out of `main` until we're ready for people to use it?  
/shrug